h1st/model/oracle/__init__.py

```diff
@@ -1,2 +1,4 @@
-from .oracle import Oracle
-from .ts_oracle import TimeSeriesOracle
+from h1st.model.oracle.oracle_modeler import OracleModeler
+from h1st.model.oracle.oracle_model import OracleModel
+from h1st.model.oracle.ts_oracle_modeler import TimeseriesOracleModeler
+from h1st.model.oracle.ts_oracle_model import TimeSeriesOracleModel
```

Comment:
Can use `TimeSeriesOracle`, leaving out `Model` which is sufficiently well implied.